lib/src/base-styles/_nx-alert.scss

```diff
@@ -66,11 +66,11 @@
 }
 
 .nx-alert--info {
-  background-color: #f1f7ff;
-  border-color: #045dd1;
+  background-color: $nx-blue-50;
+  border-color: $nx-blue-600;
 
   > .nx-icon {
-    color: #045dd1;
+    color: $nx-blue-600;
```

Comment:
So is the idea that we'll only use the semantic variables for cases where a lot of reuse is expected?

Reply:
> So is the idea that we'll only make semantic variables for cases where a lot of reuse is expected?

Yes.